### PR TITLE
New docs theme

### PR DESCRIPTION
**Pull request by ozdanborne**
## Description

Refactors #2998 to work with new branch-per-version model.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit bbbc8522b24e1e72551f8912ebcb23f3fc046aab

https://deploy-preview-3123--calico.netlify.com

File: security/policy-rules.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+title: Policy rules
+description: Learn to control traffic to/from endpoints using namespaces, service accounts, external IPs or networks, and ICMP ping in Calico network policy rules.
+show_read_time: false
+show_toc: false
+---
+
+{{ page.description }}
+
+{% capture content %}{% include index.html %}{% endcapture %}
+{{ content | replace: "    ", "" }}
```
Comment (spikecurtis) [inline security/policy-rules.md]: breadcrumbs on this and children are incorrect

File: security/hosts.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+title: Policy for hosts
+description: Use the same Calico network policy configuration for workloads to restrict traffic between hosts and the outside world. 
+show_read_time: false
+show_toc: false
+---
+
+{{ page.description }}
+
+{% capture content %}{% include index.html %}{% endcapture %}
+{{ content | replace: "    ", "" }}
```
Comment (spikecurtis) [inline security/hosts.md]: breadcrumbs for this page and children are incorrect

File: security/services.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+title: Policy for Kubernetes services
+description: Apply policy to Kubernetes node ports, and to services exposed externally as cluster IPs.
+show_read_time: false
+show_toc: false
+---
+
+{{ page.description }}
+
+{% capture content %}{% include index.html %}{% endcapture %}
+{{ content | replace: "    ", "" }}
```
Comment (spikecurtis) [inline security/services.md]: breadcrumbs for children of this page is incorrect

File: security/extreme-traffic.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+title: Policy for extreme traffic
+description: Calico network policy early in the Linux packet processing pipeline to avoid DoS attacks, and selectively bypass Linux conntrack for extremely large number of connections.
+show_read_time: false
+show_toc: false
+---
+
+{{ page.description }}
+
+{% capture content %}{% include index.html %}{% endcapture %}
+{{ content | replace: "    ", "" }}
```
Comment (spikecurtis) [inline security/extreme-traffic.md]: children of this page don't have correct breadcrumbs

File: security/istio.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+title: Policy for Istio
+description: Create standard Calico network policies with application layer specific attributes for Istio service mesh.
+show_read_time: false
+show_toc: false
+---
+
+{{ page.description }}
+
+{% capture content %}{% include index.html %}{% endcapture %}
+{{ content | replace: "    ", "" }}
```
Comment (spikecurtis) [inline security/istio.md]: children of this page have incorrect breadcrumbs

**Comment by ozdanborne** [comment]
@spikecurtis 
- I've disabled breadcrumbs for now. It seems they are using the URL of the page, not the location of the page in the navbar. I will have AlexV's team update to use sidebar once merged.
- The fact that pages are missing descriptions and thus landing pages look ugly is a known issue. The sooner we merge this, the sooner @bmckercher123 can go in and update the missing descriptions.
- I've updated all internal links to moved pages to link to the page at their new location.

**Pull request merged**